main.go

```diff
@@ -154,6 +158,10 @@ func main() {
 					Name:  "env, e",
 					Usage: "Pass an additional environment variable (new flag per variable)",
 				},
+				cli.StringSliceFlag{
+					Name:  "label, l",
+					Usage: "Add one or more docker labels to every node container of the cluster (Docker notation: `key=value`)",
```

Comment:
It's one label per flag, right? So a new flag per label, correct? Maybe we should mention that here... just in case someone tries to use e.g. a comma-separated list of labels.

Reply:
Ok. I've updated the commit.